infra/gcb/build_msan_libs.py

```diff
@@ -57,6 +58,7 @@ def main():
       projectId='oss-fuzz-base', body=build_body).execute()
   build_id = build_info['metadata']['build']['id']
 
+  print >> sys.stderr, 'Logs:', build_base_images.get_logs_url(build_id)
   print build_id
```

Comment:
maybe add 'Build ID:' to match your format.

Reply:
The same as above, but we can update all the scripts. Might be a good idea to move these little helpers into some common utils file.